main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	// "strconv" // Was removing automatically so I commented it out
```

Comment:
the 'reflect' is there for debugging purposes, and I'm not sure if we'll need the string convert yet - we can take those out if they don't end up being needed!